src/elm/Page/Profile.elm

```diff
@@ -121,452 +105,342 @@ view loggedIn model =
+viewProfileItem : String -> Html msg -> VerticalAlign -> Html msg
+viewProfileItem lbl content vAlign =
+    let
+        vAlignClass =
+            case vAlign of
+                Top ->
+                    "items-start"
 
-              else
-                div [ class "profile-img-hover" ] [ Icon.upload "" ]
-            ]
+                Center ->
+                    "items-center"
+    in
+    li
+        [ class "flex justify-between py-4"
+        , class vAlignClass
+        ]
+        [ span [ class "text-sm leading-6 mr-4" ]
+            [ text lbl ]
+        , span [ class "text-indigo-500 font-medium text-sm text-right" ]
+            [ content ]
         ]
 
 
-viewBadges : LoggedIn.Model -> Html msg
-viewBadges loggedIn =
+viewTransferButton : Shared -> Symbol -> String -> Html msg
+viewTransferButton shared symbol user =
     let
         text_ s =
-            text (t loggedIn.shared.translations s)
+            text (t shared.translations s)
     in
-    div [ class "profile-badges" ]
-        [ h4 [ class "profile-title" ]
-            [ Icon.brightness7 ""
-            , span [] [ text_ "profile.badge.title" ]
+    div [ class "mt-3 mb-2" ]
+        [ a
+            [ class "button button-primary w-full"
+            , Route.href (Route.Transfer symbol (Just user))
             ]
-        , div [ class "profile-badges-list" ]
-            [ div [ class "profile-badges-item profile-badges-item--inactive" ]
-                [ img [ src "images/badges-currency-starter.svg" ]
-                    []
-                , span [] [ text_ "profile.badge.currencyStarter" ]
-                ]
-            , div [ class "profile-badges-item" ]
-                [ img [ src "images/badge-developer.svg" ]
-                    []
-                , span [] [ text_ "profile.badge.developer" ]
-                ]
-            , div [ class "profile-badges-item" ]
-                [ img [ src "images/badge-seed-team.svg" ]
-                    []
-                , span [] [ text_ "profile.badge.seedTeam" ]
+            [ text_ "transfer.title" ]
+        ]
+
+
+viewDownloadPdfErrorModal : Model -> LoggedIn.Model -> Html Msg
+viewDownloadPdfErrorModal model loggedIn =
+    let
+        modalVisibility =
+            case model.maybePdfDownloadedSuccessfully of
+                Just isDownloaded ->
+                    if not isDownloaded then
+                        Modal.shown
+
+                    else
+                        Modal.hidden
+
+                Nothing ->
+                    Modal.hidden
+
+        privateKey =
+            case LoggedIn.maybePrivateKey loggedIn of
+                Nothing ->
+                    ""
+
+                Just pk ->
+                    pk
+
+        content =
+            div
+                []
+                [ p
+                    [ class "w-full font-medium text-heading text-2xl mb-2" ]
+                    -- I'll add the translations after we agreed about this case
```

Comment:
I'm 100% on this. This corner case happened a few times on MUDA and this will be good solution since a few users simply stopped using after facing this

Reply:
The text in the popup says that the users could contact us. How they can do that? Do we have any email which could be used for the user's support?

Reply:
lets discuss this over on #300

Reply:
for this PR alone I think just the text is enough. Thoughts @heltonlr